backend/src/core/middlewares/job.middleware.ts

```diff
@@ -19,15 +19,35 @@ const sendCampaign = async (
   try {
     const userId = req.session?.user?.id
     const { campaignId } = req.params
-    const { rate } = req.body
+    // also need to retrieve if its to be scheduled
+    const { rate, scheduledTiming } = req.body
     const logMeta = { action: 'sendCampaign', campaignId }
 
+    // convert scheduled timing to date object
+    // if passed in then convert, if not just pass empty / undefined
+    const formattedTiming = scheduledTiming
+      ? new Date(scheduledTiming)
+      : undefined
```

Comment:
`null` is a better choice for a value that's intentionally missing (`undefined` is for unintentionally missing values)

Reply:
changed